src/runtime_src/core/tools/xbmgmt2/flash/flasher.cpp

```diff
@@ -124,7 +126,14 @@ int Flasher::upgradeFirmware(const std::string& flasherType,
         XQSPIPS_Flasher xqspi_ps(m_device);
         if (primary == nullptr)
         {
-            retVal = xqspi_ps.revertToMFG();
+            std::string golden_file = getQspiGolden();
+            if (golden_file.empty()) {
+                std::cout << "ERROR: Doesn't find golden in base pkg" << std::endl;
```

Comment:
Change error message to "Golden image not found in the base package. Can't revert to golden"

Reply:
changed.